### PR TITLE
updated Sauce Labs example with current, working code

### PR DESCRIPTION
**Pull request by ShamarYarde**
### Description
The previous Sauce Labs code was updated, because the previous one is outdated and no longer works.

### Motivation
To allow users to understand how to show all their previous Sauce Labs jobs locally on their computer.

### CODE REVIEW

**Review by ShamarYarde — COMMENTED** [review]
Previously contained code which was not needed for desired results.

File: files/en-us/learn/tools_and_testing/cross_browser_testing/automated_testing/index.md [hunk 1]
```diff
@@ -430,32 +430,25 @@ Let's have a brief look at how we'd access the API using Node.js and [node-sauce
...
-     password: "your-sauce-api-key",
-   });
+   (async () => {
+     const myAccount = new SauceLabs({
+       username: "your-sauce-username",
+       password: "your-sauce-api-key",
+     });
+
+     // get full webdriver url from the client depending on region:
+     console.log(myAccount.webdriverEndpoint); // outputs "https://ondemand.us-west-1.saucelabs.com/
```
Comment (pepelsbey) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/automated_testing/index.md:442]:
```suggestion
     // Get full WebDriver URL from the client depending on region:
     console.log(myAccount.webdriverEndpoint);
```

Output will be different depending on the region. The specific output might be confusing.

File: files/en-us/learn/tools_and_testing/cross_browser_testing/automated_testing/index.md [hunk 2]
```diff
@@ -430,32 +430,25 @@ Let's have a brief look at how we'd access the API using Node.js and [node-sauce
...
-           myAccount.showJob(job.id, (err, res) => {
-             let str = `${res.id}: Status: ${res.status}`;
-             if (res.error) {
-               str += `\x1b[31m Error: ${res.error}\x1b[0m`;
-             }
-             console.log(str);
-           });
-         }
-       });
+     // get job details of last run job
```
Comment (pepelsbey) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/automated_testing/index.md:444]:
```suggestion
     // Get job details of last run job
```

**Review by pepelsbey — COMMENTED** [review]
Hey! Nice catch :) Thank you for updating the example. I’d probably adjust it a bit to fit MDN better. Otherwise, it looks great!

**Review by pepelsbey — APPROVED** [review]
Thank you :)

**Pull request merged**